src/scss/grommet-core/_objects.search-input.scss

```diff
@@ -1,53 +1,54 @@
 // (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP
 
+.grommet .search-input,
```

Comment:
The .grommet input style overwrites the right-padding that's used to prevent overlap of the icon and input text. Do you have a recommendation for another way to do this? Thanks.